pkg/storaged/fsys-is-empty.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+set -eux
+
+dev=$1
+
+need_unmount=""
+mp=$(findmnt -no TARGET "$dev" | cat)
```

Comment:
That cat seems unnecessary:

```
[jelle@t14s][~/projects/cockpit/main]%x=$(findmnt -no TARGET /dev/nvme0n1p1)
[jelle@t14s][~/projects/cockpit/main]%echo $x
/boot
```

Reply:
It protects against the exit code of `findmnt`. When $dev is not mounted, findmnt exits with code 1, which would stop the script because of `set -e`.  In a pipeline, the last sub-command determines the exit code of the while pipeline, and `cat` always exits with 0.